test/contrib/aws/instrumentation_test.rb

```diff
@@ -0,0 +1,68 @@
+require 'helper'
+require 'aws-sdk'
+require 'ddtrace'
+require 'ddtrace/contrib/aws/patcher'
+require 'ddtrace/ext/http'
+
+module Datadog
+  module Contrib
+    module Aws
+      class InstrumentationTest < Minitest::Test
+        def setup
+          # initializes the tracer and a stub client
+          @tracer = enable_tracer
+          @client = ::Aws::S3::Client.new(stub_responses: true)
+        end
+
+        def test_pin_defaults
```

Comment:
@p-lambert always test our pin defaults, so we know if we break defaults while changing how the integration works